code/datums/mutations/hulk.dm

```diff
@@ -11,6 +11,10 @@
 /datum/mutation/human/hulk/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
+	if(jobban_isbanned(owner, ROLE_HULK))
```

Comment:
Matter of fact, what is this hardcoded shit?

Go home, and write a proper genetics jobban for this. Don't half-ass this.